docs/source/what_is_jupyter.md

```diff
@@ -0,0 +1,412 @@
+you can imagine. You can invent new editing and viewing experiences for your
+data this way, using the interactive computing capabilities provided by
+the kernels.
+
+## Benefits of a many-piece design
+
+By breaking up a program like JupyterLab into multiple component pieces, you
+can customize the software to meet your needs. If one piece is missing something,
+you can replace it with a custom version made by yourself or another person to
+add whatever features you would like to see.
+
+You can also invent completely new experiences using those pieces that the
+designers may not have imagined when they started, and they can often inter-
+operate seamlessly with existing Jupyter software.
+
+Because Jupyter Server provides kernel communication and management features
+in a cohesive, self-contained package, for instance, new notebook editor
+programs can focus solely on adding new interface and editing experiences,
+leaving the task of creating and managing new kernels completely up to the
+Jupyter Server.
+
+JupyterCAD is a 3D modeling tool built on top of Jupyter software, for
+instance, and there are other examples too (LINK HERE).
+
+## Talking to Kernels (The Jupyter Protocol)
+
+Anyone can make new software that talks to Jupyter's kernels (read more
+about those above), by using the Jupyter Protocol. The protocol provides
+a standardized blueprint for passing information back and forth between
+the kernels and other software.
+
+By implementing the designs described in the Jupyter Protocol, you could
+invent a completely new interactive programming experience, or add support
+for a new programming language to Jupyter.
+
+Because Project Jupyter is free and open, it encourages anyone to explore
+new ways of working with their notebooks and kernels, and likes to offer
+compatibility and interoperation with other software.
+
+TODO add more here? examples? etc.
+
+# What else should I know about Project Jupyter?
+
+Jupyter software is free and open-source, developed by a global community
+of volunteers and contributors, available for the benefit of all.
+
+Project Jupyter welcomes people from all backgrounds and with many types
+of skills (not just software!) so we encourage you to join us!
+
+You can share feedback about your experiences directly with the people who
+make Jupyter software, or volunteer and contribute to help Jupyter in many
+different ways, like:
+
+- Writing tutorials
+- Testing newly released versions
+- Adding software features
+- Hosting the weekly video meetings (LINK)
+- Helping others in the community
+- ...and more!
+
+The project is split into largely independent subprojects which handle
+different aspects of Jupyter software and the community. A central council,
+the Jupyter Executive Council (LINK), makes decisions about project-wide
+goals and policies, while different subprojects handle the actual development
+of the various software components.
```

Comment:
This may be too much detail for an introduction. Might be best to link to the governance docs, or perhaps to delete this entirely.